specification/batch/Azure.Batch/models.tsp

```diff
@@ -2145,6 +2231,9 @@ model BatchDiffDiskSettings {
 
 @doc("Specifies how Tasks should be distributed across Compute Nodes.")
 model BatchTaskSchedulingPolicy {
+  @doc("The order for scheduling tasks from different jobs with the same priority. If not specified, the default is none.")
+  jobDefaultOrder?: BatchJobDefaultOrder;
```

Comment:
look into if we can set a default for this enum

Reply:
Note that `= BatchJobDefaultOrder.None` would mean that this be a server-side default. Meaning, upon PUT and user not set the property, client also won't send the property. Server would assume the "none" and give back this "none" in property upon GET